pyaerocom/aeroval/coldatatojson_engine.py

```diff
@@ -165,114 +204,157 @@ def process_coldata(self, coldata: ColocatedData):
         if annual_stats_constrained:
             data = _apply_annual_constraint(data)
 
-        if not diurnal_only:
-            logger.info("Processing statistics timeseries for all regions")
-            input_freq = self.cfg.statistics_opts.stats_tseries_base_freq
-
-            for reg in regnames:
-                try:
-                    stats_ts = _process_statistics_timeseries(
-                        data=data,
-                        freq=main_freq,
-                        region_ids={reg: regnames[reg]},
-                        use_weights=use_weights,
-                        use_country=use_country,
-                        data_freq=input_freq,
+        if not "just_for_viz" in coldata.data.attrs:  # make the regular json output
```

Comment:
Unfortunately not, because `coldata.data` is a netCDF file, and one can not store a boolean as a netCDF attribute

Reply:
could it be zero?

Reply:
No because the implementation only adds this attribute for the `colocateddata_for_profile_viz` part of `ColocatedDataLists` in `colocation_3d.py`. In theory nothing is stopping one adding `"just_for_viz" = 0` to `colocateddata_for_statistics`, but there isn't really any point in doing so, and adding it would be an unnecessary attribute. I will add a check there to make sure it is 1, though.